third_party/packages/cupertino_icons/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dev_dependencies:
     sdk: flutter
   flutter_test:
     sdk: flutter
-  path: 1.9.0
+  path: any
```

Comment:
Please don't ever use `any` anywhere in a package; this is a recipe for OOB tree breakage.